gpt_engineer/core/ai.py

```diff
@@ -354,6 +357,15 @@ def _create_chat_model(self) -> BaseChatModel:
                 max_tokens=4096,  # vision models default to low max token limits
             )
 
+        if "claude" in self.model_name:
+            return ChatAnthropic(
+                model=self.model_name,
+                temperature=self.temperature,
+                callbacks=[StreamingStdOutCallbackHandler()],
+                streaming=True,
+                max_tokens_to_sample=4096,
+            )
+
```

Comment:
This is duplicated from the block below (except for the streaming).

Fix in https://github.com/gpt-engineer-org/gpt-engineer/pull/1100